### ISSUE TITLE
Passing options does no longer work

### ISSUE THREAD

**Issue by GXGOW** [gort818/qtwebflix#20]
I think this has something to do with your fix for #19, applied on 5d4001b58feec7a59986a28bb7a44b8f8aff702c. I've installed the update from the AUR package today, and since then qtwebflix does not recognize any of the parameters I give to it:
```bash
$ qtwebflix --register-pepper-plugins="/usr/lib/qt/plugins/ppapi/libwidevinecdmadapter.so; application/x-ppapi-widevine-cdm"
Unknown option 'register-pepper-plugins'.
```

Of course, launching without parameters still works without issues, but then the widevine libraries won't be loaded.

**Comment by gort818** [comment]
Thanks for the issue, I will take a look in a bit.

The widevine libraries will be loaded  without the command line option as long as chromium is not installed.

**Comment by GXGOW** [comment]
> as long as chromium is not installed.

Chromium is my main browser, so I guess I'm out of luck. Oh well, it's not that bad. Thanks for looking into it!

**Comment by gort818** [comment]
I have not looked into it yet, I will in a few hours.

Are you using a DE? does  qtwebflix work when launching from the desktop shortcut?

**Comment by Surrogard** [comment]
+1
I have an ArchLinux (SwagArch to be exact) and Chromium installed.

**Comment by GXGOW** [comment]
Launching the desktop shortcut was the first thing I tried, but that didn't work unfortunately. That's why I tried launching it from the command line with the same command as in the .desktop file, so I could see possible errors.
That's where I noticed that none of the options are recognized.

**Comment by gort818** [comment]
Oops must have a made a mistake, sorry about that :grinning: 

It will be fixed today.

**Comment by gort818** [comment]
Command line parser removed.. we are up and running again.

Ok adding a command line parser removes all default qtwebengine command line options.. oops.. I dont not want to add them manually so I will find another way to fix #19.

Thanks for all the comments guys sorry about that!
